app/src/components/notifications/LNotification.vue

```diff
@@ -51,4 +96,27 @@ const show = ref(true);
             </div>
         </div>
     </div>
+
+    <!-- Banner -->
+    <div
+        v-if="show && notification.type == 'banner'"
+        class="inset-x-0 top-0 z-50 text-zinc-900"
+        :class="colour"
+    >
+        <div class="flex items-center justify-between px-6 py-1">
+            <div class="flex items-center gap-2">
+                <component :is="icon" class="h-5 min-h-5 w-5 min-w-5" />
+                <div class="flex flex-col md:inline-block md:align-middle">
+                    <span class="text-md md:text-sm">{{ notification.title }}</span>
+                    <span v-if="notification.description" class="text-xs md:ml-3">{{
+                        notification.description
+                    }}</span>
+                </div>
+            </div>
+            <XMarkIcon
+                @click="notification.id ? removeNotification(notification.id) : (show = false)"
```

Comment:
Similar to the comment above, this will always call `removeNotification`. I am wondering if that is right here as well. The banner is still valid, but the user is hiding it. Especially in the case of the offline banner, should the user be able to click it away? It serves as a nice warning to them why they can't play videos for example, which could 
be confusing otherwise. And most people are conditioned to click away banners without reading them.

So I would propose to remove the close button for the banner, or at least make it optional.